### DISCUSSION TITLE
trying-all-the-things

### DISCUSSION THREAD

**Discussion by giscus** [SethBarberee/sethbarberee.github.io#12] [Announcements]
# trying-all-the-things

This is mainly to try out the blog feature of Jekyll. I’m slowly learning how this works so bear in mind as this is also my first post. So with that said, I hope to write more posts in the future with more entertaining things such as current projects.

https://sethbarberee.github.io/trying-all-the-things

**Comment by SethBarberee** [comment]
Testing giscus to verify things are working
